inst/tutorials/r_and_rstudio_intermediate/r_and_rstudio_intermediate.Rmd

```diff
@@ -0,0 +1,139 @@
+---
+title: "Intermediate R and RStudio"
+author: "Cathy Yan"
+date: "version `r format(Sys.time(), '%B %d, %Y')`"
+output:
+  learnr::tutorial:
+    progressive: true
+    allow_skip: true
+runtime: shiny_prerendered
+description: [DESCRIPTION]
+---
+
+```{r setup, include = FALSE}
+# General learnr setup
+library(learnr)
+knitr::opts_chunk$set(echo = TRUE)
+library(educer)
+# Helper function to set path to images to "/images" etc.
+setup_resources()
+
+# Tutorial specific setup
+library(dplyr)
+library(readr)
+library(tidyverse)
+```
+
+## Learning objectives
+
+Here's what you'll learn from each section of this tutorial:
+
+Loops in R:
+
+- Explain the mechanisms of for and while loops
+- Implement for and while loops
+- Recognize and understand how nested loops work
+
+Conditionals in R:
+
+- Write and evaluate and/or statements
+- Identify the components of if/else statements
+- Implement conditionals within loops
+
+Functions in R:
+
+- Write functions that operate on arguments
+- Understand the role of helper functions
+- Implement helper functions
+
+Manipulating Data Frames:
+
+- Implement the pipe operator
+- Use the `select()`, `filter()`, and `mutate()` functions appropriately
+- Operate on individual columns within a data frame
+- Change the data type of columns in a data frame
+- Replace NAs in a vector or data frame
+
+Other ideas: Tidying data, joining data frames, loading in data from an external source
+
+## Review
+
+Before proceeding, complete the five questions below reviewing content from Introduction to R and RStudio Fundamentals. If any of the concepts seem unfamiliar, please revisit the previous tutorial.
+
+```{r using-libraries, echo=FALSE}
+quiz(question("In no particular order, which actions are required before using functions from a new package?",
+      answer("Install the package", correct=TRUE),
+      answer("Download vignettes"),
+      answer("Load the package", correct=TRUE),
+      answer("Install dependencies"))
+)
+```
+
+Pull up the help page for the function `quantile()`.
+
+```{r q-help, exercise=TRUE}
+# your code here
+```
+
+```{r q-help-solution}
+?quantile
+```
+
+```{r review, echo=FALSE}
+quiz(
+  question("What types of arguments can be passed to `quantile()`?",
```

Comment:
Arguments may be difficult to follow for new users. The probs argument has a seq  function that was not covered in the workshop